rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/RmqSourceReplicator.java

```diff
@@ -146,33 +122,51 @@ public void resume() {
 
     public List<KeyValue> taskConfigs() {
       
-        if (started && configValid) {
+        if (configValid) {
+
+            boolean adminStarted = false;
+            RPCHook rpcHook = null;
+            DefaultMQAdminExt defaultMQAdminExt = new DefaultMQAdminExt(rpcHook);
+            defaultMQAdminExt.setNamesrvAddr(this.replicatorConfig.getString(ConfigDefine.CONN_SOURCE_RMQ));
+            defaultMQAdminExt.setInstanceName(Utils.createGroupName(ConstDefine.REPLICATOR_ADMIN_PREFIX));
```

Comment:
It was changed back here。
this.defaultMQAdminExt.setAdminExtGroup(Utils.createGroupName(ConstDefine.REPLICATOR_ADMIN_PREFIX));
            this.defaultMQAdminExt.setInstanceName(Utils.createInstanceName(this.config.getString(ConfigDefine.CONN_SOURCE_RMQ)));